src/librustdoc/html/static/main.js

```diff
@@ -342,6 +342,18 @@
                 }
             }
 
+            function findArg(obj, val) {
+                if (!obj || !obj.type || obj.type.inputs.length === 0) {
```

Comment:
Heh, that's such a _negative_ way to write this ;) How about

```js
obj && obj.type && obj.type.inputs && obj.type.inputs.length > 0
```

Reply:
I wanted to avoid putting all my code in an `if` condition but either way is fine for me so let's go for it!